common-rendering/src/components/FirstPublished.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from "@emotion/react";
+import { timeAgo } from "@guardian/libs";
+import { neutral, space } from "@guardian/src-foundations";
+import { textSans } from "@guardian/src-foundations/typography";
+
+// TODO: update this code to use shared version when it is available
+const padString = (time: number) => (time < 10 ? `0${time}` : time);
+
+const FirstPublished = ({
+	firstPublished,
+	blockLink,
+}: {
+	firstPublished: number;
+	blockLink: string;
+}) => {
+	const publishedDate = new Date(firstPublished);
+	return (
+		<a
+			href={blockLink}
+			data-ignore="global-link-styling"
+			// title={publishedDate.toLocaleString()}
```

Comment:
tiny q: I can see we had this same comment in liveblock too, do we know if we need to retain it?

Reply:
Good question. @lucymonie do you know why this line was commented out?

Reply:
I don't know @marjisound, but I think it can come out as it's not in use.